airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1303,12 +1295,6 @@ definitions:
           - ["data", "records"]
           - ["data", "{{ parameters.name }}"]
           - ["data", "*", "record"]
-      decoder:
```

Comment:
Just noting that this one seems like a breaking change as there is more than one value but from my understanding even if something else than JsonDecoder was defined, we would not consider it, right? 

If so, my concern is only if an additional property would break the parsing

Reply:
Previously, the `SimpleRetriever` would set the decoder as `JsonDecoder` if the connector dev did not specify a decoder, which would then be passed down all the way to the `DpathExtractor`. The `DpathExtractor` previously allowed connector-dev-specified decoders, but the decoder that was injected via the `SimpleRetriever` would take precedent, so the connector-dev-specified one was never actually used (unless there was something like a custom `Retriever` that didn't initialize a default decoder).

Now I have removed that logic and the decoder either uses the injected decoder from the `SimpleRetriever` or initializes a `JsonDecoder`

This shouldn't be a breaking change based on my previous response.

Reply:
[DpathExtractor schema link.](https://github.com/airbytehq/airbyte/blob/3ae14bb6c0936efbf811e3aca43eb73cb1743092/airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml#L1282-L1314)